frontend/Makefile

```diff
@@ -47,11 +45,10 @@ build-server:
 build-client:
 	./node_modules/.bin/esbuild \
 		--bundle \
-		--loader:.jpg=file \
-		--loader:.png=file \
-		--loader:.svg=file \
-		--public-path=/dist \
+		--external:*.jpg \
 		--outfile=dist/client.entry.js \
 		--minify \
 		--keep-names \
+		--define:process=\{\} \
+		--define:process.env=\{\} \
```

Comment:
I can't follow why this is happening as a result of these changes? Is this unrelated or am I missing something?